tools/codegen/codespec/api_to_provider_spec_mapper_test.go

```diff
@@ -24,327 +24,318 @@ type convertToSpecTestCase struct {
 }
 
 func TestConvertToProviderSpec(t *testing.T) {
-	testCases := map[string]convertToSpecTestCase{
-		"Valid input": {
-			inputOpenAPISpecPath: "testdata/api-spec.yml",
-			inputConfigPath:      "testdata/config-no-schema-opts.yml",
-			inputResourceName:    "test_resource",
+	tc := convertToSpecTestCase{
+		inputOpenAPISpecPath: "testdata/api-spec.yml",
```

Comment:
[nit] will there ever be more than 1 api-spec? Maybe a global constant

Reply:
agree, extracted